optimade/server/config.py

```diff
@@ -169,6 +169,12 @@ def load_from_ini(self):
         else:
             self.provider = self._DEFAULTS("provider")
 
+        self.aliases = {}
+        for endpoint in {"links", "references", "structures"}:
+            if f"{endpoint}.aliases" in config.sections():
+                section = config[f"{endpoint}.aliases"]
+                self.aliases[endpoint] = tuple((key, section[key]) for key in section)
```

Comment:
```suggestion
                self.aliases[endpoint] = tuple(section.items())
```
You could remove the previous line as well and simply put it directly in this line instead of `section`.

Reply:
Note that both methods will return the sections key and values as strings with all non-capitalized characters.